cluster/gce/gci/configure.sh

```diff
@@ -245,7 +245,7 @@ function install-node-problem-detector {
       local -r npd_version="${DEFAULT_NPD_VERSION}"
       local -r npd_hash="${DEFAULT_NPD_HASH}"
   fi
-  local -r npd_tar="node-problem-detector-${npd_version}.tar.gz"
+  local -r npd_tar="node-problem-detector-${npd_version}-${HOST_PLATFORM}_${HOST_ARCH}.tar.gz"
```

Comment:
does the hard-coded hash work against a dynamic platform/arch download?

Reply:
😰Only linux tar hash, I thought.

Forget to check the hash for windows, and I need verify. Thanks for reminding

Reply:
I guess not. So for ARM you would need to override the hash via the NODE_PROBLEM_DETECTOR_TAR_HASH variable. Not great

Reply:
I think all the default hashes are for linux/amd64,  including CRICTL, CNI, etc.

Reply:
Not an issue for NPD 0.8.8. Created https://github.com/kubernetes/kubernetes/pull/103387 for 0.8.9 to not forget about it

Reply:
https://github.com/kubernetes/node-problem-detector/releases/tag/v0.8.9 is released.
no urgent fix. we may upgrade it later.